nosqlbench/http-jsonapi-crud-advanced.yaml

```diff
@@ -0,0 +1,301 @@
+        X-Cassandra-Token: "<<auth_token:my_auth_token>>"
+        Content-Type: "application/json"
+        ok-body: ".*\"ok\":1.*"
+        body: |
+          {
+            "deleteCollection": {
+              "name": "<<collection:docs_collection>>"
+            }
+          }
+
+      create-collection:
+        method: POST
+        uri: <<protocol:http>>://{weighted_hosts}:<<jsonapi_port:8080>><<path_prefix:>>/v1/<<namespace:jsonapi_crud_basic>>
+        Accept: "application/json"
+        X-Cassandra-Request-Id: "{request_id}"
+        X-Cassandra-Token: "<<auth_token:my_auth_token>>"
+        Content-Type: "application/json"
+        ok-body: ".*\"ok\":1.*"
+        body: |
+          {
+            "createCollection": {
+              "name": "<<collection:docs_collection>>"
+            }
+          }
+
+  write:
+    ops:
+      # aka insertOne
+      write-document:
+        space: "{space}"
+        method: POST
+        uri: <<protocol:http>>://{weighted_hosts}:<<jsonapi_port:8080>><<path_prefix:>>/v1/<<namespace:jsonapi_crud_basic>>/<<collection:docs_collection>>
+        Accept: "application/json"
+        X-Cassandra-Request-Id: "{request_id}"
+        X-Cassandra-Token: "<<auth_token:my_auth_token>>"
+        Content-Type: "application/json"
+        ok-body: ".*\"insertedIds\":\"{seq_key}\".*"
+        body: |
+          {
+            "insertOne" : {
+              "document" : {
+                "_id" :         "{seq_key}",
+                "user_id":      "{user_id}",
+                "created_on":   {created_on},
+                "gender":       "{gender}",
+                "full_name":    "{full_name}",
+                "married":      {married},
+                "address": {
+                  "primary": {
+                      "city":   "{city}",
+                      "state":  "{state}",
+                      "cc":     "{country_code}"
+                  },
+                  "secondary":  {}
+                },
+                "coordinates": [
+                                {lat},
+                                {lng}
+                ],
+                "children":     [],
+                "friends": [
+                                "{friend_id}"
+                ]
+              }
+            }
+          }
+
+  read-single:
+    ops:
+      # aka findOne with _id as filter
+      read-document:
+        space: "{space}"
+        method: POST
+        uri: <<protocol:http>>://{weighted_hosts}:<<jsonapi_port:8080>><<path_prefix:>>/v1/<<namespace:jsonapi_crud_basic>>/<<collection:docs_collection>>
+        Accept: "application/json"
+        X-Cassandra-Request-Id: "{request_id}"
+        X-Cassandra-Token: "<<auth_token:my_auth_token>>"
+        Content-Type: "application/json"
+        ok-body: ".*\"data\".*"
+        body: |
+          {
+            "findOne" : {
+              "filter" : {
+                "_id" : "{random_key}"
+              }
+            }
+          }
+
+  read-multiple:
```

Comment:
if you join different types or reads in this `read-multiple`, will we be able to see individual results of `read-sorted` for example? I guess so, but just worth of checking..